app/services/metric_service.rb

```diff
@@ -0,0 +1,58 @@
+class MetricService
+  class << self
```

Comment:
Might be good to get https://github.com/alphagov/email-alert-api/blob/master/app/services/email_sender_service.rb#L12 in here too.

Reply:
added